services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -482,6 +487,22 @@ private RouteOptions routeOptions() {
       .user(DirectionsCriteria.PROFILE_DEFAULT_USER)
       .enableRefresh(true)
       .metadata(true)
+      .experimental(
+        RouteOptions.Experimental.builder()
+          .engine("electric")
+          .evInitialCharge(80000)
+          .evMaxCharge(80000)
+          .evConnectorTypes("ccs_combo_type1,ccs_combo_type2")
+          .energyConsumptionCurve("0,300;20,160;80,140;120,180")
+          .auxiliaryConsumption("auxiliaryConsumption")
```

Comment:
The auxilliare consumption is currently not prioritized as that information can be passed through overloaded via the charging-curve parameter https://github.com/mapbox/nav-api-intake/issues/893#issuecomment-969241997

Reply:
Would you recommend removing this param for now?